src/libponyrt/sched/start.c

```diff
@@ -148,6 +152,17 @@ PONY_API int pony_init(int argc, char** argv)
     exit(0);
   }
 
+  if (opt.noscale)
```

Comment:
Based on the description on the issue, I think this shouldn't take precedence… yet dunno how I can make sure it actually was explicitly set (with `0` being the default currently). Also what should it do it `opt.min_threads == opt.threads`, as that'd be the equivalent of `opt.noscale`…